gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -98,12 +105,22 @@ public synchronized void shutdown() {
 
   @Override
   public boolean isShutdown() {
-    return isTransportShutdown;
+    boolean isShutdown = deadlineScheduledExecutorService.isShutdown();
+    // Check that the Gax's ExecutorService is shutdown as well
+    if (usingDefaultExecutor) {
+      isShutdown = isShutdown && ((ExecutorService) executor).isShutdown();
+    }
+    return isShutdown;
```

Comment:
I think we still need to take `isTransportShutdown` into consideration because `httpTransport.shutdown()` could fail even if `deadlineScheduledExecutorService.shutdown()` succeeded.

Reply:
Ah, makes sense. I'll add that logic in again. Thanks!